gammapy/data/tests/test_data_store.py

```diff
@@ -150,3 +151,38 @@ def test_make_psf(pars, result):
     assert_quantity_allclose(psf.energy[10], result["psf_energy"])
     assert_quantity_allclose(psf.exposure[10], result["psf_exposure"])
     assert_quantity_allclose(psf.psf_value[10, 50], result["psf_value"])
+
+
+def test_make_meanedisp(tmpdir):
```

Comment:
You have to mark this test
```
@requires_data('gammapy-extra')
```
https://travis-ci.org/gammapy/gammapy/jobs/175732342#L1722

And if it uses scipy or some other optional dependency, also
```
@requires_dependency('scipy')
```

Also, please rename: `test_make_meanedisp` -> `test_make_mean_edisp`